client/container.go

```diff
@@ -106,3 +106,14 @@ func (client *APIClient) ContainerStartExec(execid string, config *types.ExecSta
 
 	return client.hijack("/exec/"+execid+"/start", url.Values{}, config, header)
 }
+
+//ContainerRename rename a container.
+func (client *APIClient) ContainerRename(name string, newName string) error {
+	q := url.Values{}
+	q.Add("newName", newName)
```

Comment:
In swagger.yml, you add a query which is named `name`, while here the query name is `newName`. Please make sure the name is consistent.